pom.xml

```diff
@@ -343,8 +343,8 @@
                         </oldVersion>
                         <parameter>
                             <onlyModified>true</onlyModified>
-                            <breakBuildOnBinaryIncompatibleModifications>true</breakBuildOnBinaryIncompatibleModifications>
-                            <breakBuildBasedOnSemanticVersioning>true</breakBuildBasedOnSemanticVersioning>
+                            <breakBuildOnBinaryIncompatibleModifications>false</breakBuildOnBinaryIncompatibleModifications>
+                            <breakBuildBasedOnSemanticVersioning>false</breakBuildBasedOnSemanticVersioning>
```

Comment:
You don't want to disable this

Reply:
I did it for a successful build. I'll revert it later

Reply:
Please create a followup JIRA item for this so we dont forget.